README.md

```diff
@@ -93,32 +94,30 @@ The same test could be written as below, using the protractor-helper library.
 ```js
 const protractorHelper = require("protractor-helper");
 
-describe("Sign up page", () => {
+describe("Sign up page", async () => {
   it("successful sign up", () => {
-    browser.get("https://example.com/sign-up");
+    await browser.get("https://example.com/sign-up");
 
     const emailField = element(by.id("email"));
     const passwordField = element(by.id("password"));
     const signupButton = element(by.id("signup"));
 
-    protractorHelper.fillFieldWithTextWhenVisible(
-      emailField,
-      "valid@email.com"
-    );
-    protractorHelper.fillFieldWithTextWhenVisible(
-      passwordField,
-      "validpassword"
-    );
-    protractorHelper.clickWhenClickable(signupButton);
+    await protractorHelper.fillFieldWithTextWhenVisible(emailField, "valid@email.com");
+    await protractorHelper.fillFieldWithTextWhenVisible(passwordField, "validpassword");
+    await protractorHelper.clickWhenClickable(signupButton);
 
     const avatar = element(by.id("avatar"));
 
-    protractorHelper.waitForElementVisibility(avatar);
+    await protractorHelper.waitForElementVisibility(avatar);
   });
 });
 ```
 
-As you can see, by using the protractor-helper library the code is easier to read. Also, there is no need of unnecessary complexity.
+As you can see, by using the protractor-helper library the code is shorter and easier to read. Also, there is no need of unnecessary complexity.
+
+## Pre-requirements
+
+- Node.js v8+ (due to the use of async/await)
```

Comment:
Protractor version 6.0.0 isn't also a pre-requirement?

Reply:
I've executed the existing tests after the changes with version 5.4.2 of Protractor, and all tests have still passed, so I don't think we need that.

Reply:
Please resolve this conversation if you are ok with that.